pins_arduino.h not found

### No pins_arduino.h in the variants
I am using red board Artemis nano with 2.8" adafruit TFT LCD shield. For simple graphic test, I am getting error 
`pins_arduino.h: No such file or directory`

###  workbench
* I am using Arduino IDE version 1.8.9
* The Arduino is connected as shown here: 
[https://learn.adafruit.com/adafruit-2-8-tft-touch-shield-v2/connecting](https://learn.adafruit.com/adafruit-2-8-tft-touch-shield-v2/connecting)

### Expected behaviour
The LCD screen should show graphical outputs

### Actual behaviour

`Arduino: 1.8.9 (Windows Store 1.8.21.0) (Windows 10), Board: "SparkFun RedBoard Artemis Nano, 115200, Ambiq Secure Bootloader (Advanced)"`

`C:\Users\Documents\Arduino\libraries\Adafruit_ILI9341\Adafruit_ILI9341.cpp:52:12: fatal error: pins_arduino.h: No such file or directory`
  ` #include "pins_arduino.h"`
         `   ^~~~~~~~~~~~~~~~`
`compilation terminated.`

--- a/cores/arduino/ard_sup/Arduino.h
+++ b/cores/arduino/ard_sup/Arduino.h
@@ -41,6 +41,8 @@ extern "C"
 #include "am_util.h"
     // #include "am_bsp.h"
 
+#include "hooks.h"
+
 #include <math.h> //Gets us pow()
 
 #ifdef ARDUINO_REDEFINE_OVERFLOW

--- a/cores/arduino/ard_sup/ard_supers/hooks.c
+++ b/cores/arduino/ard_sup/ard_supers/hooks.c
@@ -19,7 +19,19 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 */
-#ifndef _PINS_ARDUINO_H_
-#define _PINS_ARDUINO_H_
 
-#endif // _PINS_ARDUINO_H_
+#include "hooks.h"
+
+/**
+ * Empty yield() hook.
+ *
+ * This function is intended to be used by library writers to build
+ * libraries or sketches that supports cooperative threads.
+ *
+ * Its defined as a weak symbol and it can be redefined to implement a
+ * real cooperative scheduler.
+ */
+static void __empty() {
+	// Empty
+}
+void yield(void) __attribute__ ((weak, alias("__empty")));

--- a/cores/arduino/ard_sup/ard_supers/hooks.h
+++ b/cores/arduino/ard_sup/ard_supers/hooks.h
@@ -1,0 +1,27 @@
+/*
+Copyright (c) 2019 SparkFun Electronics
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+*/
+#ifndef _HOOKS_H_
+#define _HOOKS_H_
+
+void yield( void ) ;
+
+#endif // _HOOKS_H_

--- a/cores/arduino/ard_sup/ard_supers/pins_arduino.h
+++ b/cores/arduino/ard_sup/ard_supers/pins_arduino.h
@@ -1,0 +1,27 @@
+/*
+Copyright (c) 2019 SparkFun Electronics
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+*/
+#ifndef _PINS_ARDUINO_H_
+#define _PINS_ARDUINO_H_
+
+// This file exists solely for compilation compatibility with legacy libraries
+
+#endif // _PINS_ARDUINO_H_

--- a/cores/arduino/ard_sup/ard_supers/wiring_private.h
+++ b/cores/arduino/ard_sup/ard_supers/wiring_private.h
@@ -1,0 +1,27 @@
+/*
+Copyright (c) 2019 SparkFun Electronics
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+*/
+#ifndef _WIRING_PRIVATE_H_
+#define _WIRING_PRIVATE_H_
+
+// This file exists solely for compilation compatibility with legacy libraries
+
+#endif // _WIRING_PRIVATE_H_

